src/ucp/core/ucp_ep.c

```diff
@@ -933,8 +934,9 @@ void ucp_ep_config_lane_info_str(ucp_context_h context,
         p += strlen(p);
     }
 
-    if (lane == key->rndv_lane) {
-        snprintf(p, endp - p, " zcopy_rndv");
+    prio = ucp_ep_config_get_rma_prio(key->rndv_lanes, lane);
```

Comment:
need to rename this function and remove the name "rma" from it since "rma" referes only to put/get operations and not rndv. new name suggestion: ucp_ep_config_get_multi_lane_prio()

Reply:
renamed